tests/__init__.py

```diff
@@ -0,0 +1,2 @@
+import sys
```

Comment:
Don't do that in code (as suggested above).

Reply:
Thank you, now removed.